src/index.tsx

```diff
@@ -225,12 +225,30 @@ class UserJourneyTracker {
     public init(isLoggedIn?: boolean, userId?: string): void {
         if (this.isInitialized) return;
 
-        // Set login state if provided
+        // Check login state from cookies for static websites
+        if (typeof window !== 'undefined') {
+            const loginCookie = this.getCookie('client_information');
+            if (loginCookie) {
+                try {
+                    const clientInfo = JSON.parse(loginCookie);
+                    if (clientInfo && clientInfo.is_logged_in) {
+                        this.isLoggedIn = true;
+                        if (clientInfo.user_id) {
+                            this.derivUserId = clientInfo.user_id;
+                        }
+                    }
+                } catch (e) {
+                    // Invalid cookie JSON, ignore
+                }
+            }
+        }
+
+        // Set login state if provided (overrides cookie)
         if (isLoggedIn !== undefined) {
             this.isLoggedIn = isLoggedIn;
         }
```

Comment:
What happens if the login cookie already exists, but the user calls `init()` with `isLoggedIn` set to `false`?
In that case, `this.isLoggedIn` would be incorrectly overwritten and set to `false`.
To prevent this, we should update the logic to include an else clause after `if (loginCookie)` and ensure this block runs only if the cookie is not already set.